Core.DomainModel/GDPR/DataProcessingRegistrationOversightDate.cs

```diff
@@ -1,11 +1,11 @@
 ﻿using System;
-using System.Collections;
 
 namespace Core.DomainModel.GDPR
 {
-    public class DataProcessingRegistrationOversightDate
+    public class DataProcessingRegistrationOversightDate : IHasUuid
```

Comment:
I can see this is the pattern used in Kitos, but struggle to see why the UUID's are not prevalent on all Entity objects

Reply:
It is to allow certain types of filtering (and possibly other operations).

For example we have a query method QueryByUuid which accepts an IQueryable<T> where T : class, IHasUuid.
So we can easily apply this filter to all entities with a uuid